src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetVerbCommand.cs

```diff
@@ -39,45 +39,9 @@ public string[] Group
         /// </summary>
         protected override void ProcessRecord()
         {
-            Type[] verbTypes = new Type[] { typeof(VerbsCommon), typeof(VerbsCommunications), typeof(VerbsData),
-                typeof(VerbsDiagnostic), typeof(VerbsLifecycle), typeof(VerbsOther), typeof(VerbsSecurity) };
-
-            Collection<WildcardPattern> matchingVerbs = SessionStateUtilities.CreateWildcardsFromStrings(
-                            this.Verb,
-                            WildcardOptions.IgnoreCase
-                        );
-
-            foreach (Type type in verbTypes)
+            foreach (VerbInfo verb in Verbs.GetVerbsByGroup(Verb, Group))
```

Comment:
Can we have better name for GetVerbsByGroup()? Really the method filters both by Verb and by Group.

Reply:
@iSazonov I have renamed to `FilterByVerbsAndGroups` Let me know what you think 🙂